integration_test/support/schemas.exs

```diff
@@ -388,3 +388,19 @@ defmodule Ecto.Integration.ArrayLogging do
     timestamps()
   end
 end
+
+defmodule Ecto.Integration.Bitstring do
+  @moduledoc """
+  This module is used to test:
+
+    * Bitstring type
+
+  """
+  use Ecto.Integration.Schema
+
+  schema "bitstrings" do
+    field :bs,  :bitstring
+    field :bs_with_default, :bitstring, default: <<42::6>>
```

Comment:
```suggestion
    field :bs_with_default, :bitstring
```
The reason to remove this is because it will make Ecto populate the value in the schema before reaching the database. So if we want to test the default functionality added to the adapter, we have to leave this blank.

Reply:
Right, I missed it. Does it make sense to test the `field` default value moving it to another field?

Something like this:

```elixir
  schema "bitstrings" do
    field :bs,  :bitstring
    field :bs_with_default, :bitstring
    field :bs_with_size, :bitstring, default: <<42::10>>
  end
```

or is it superfluous?

Reply:
it's alright to leave it out. it's essentially the same as adding the field yourself. so since we are testing that out already we are good